pkg/chart/gcp.go

```diff
@@ -171,33 +171,45 @@ func deleteCloudSQLInstance(ctx context.Context, instanceName, gcpProject string
 	return nil
 }
 
-func createCloudSQLInstance(ctx context.Context, dbInstance, gcpProject, gcpRegion string) error {
+func createCloudSQLInstance(ctx context.Context, teamSlug, dbInstance, gcpProject, gcpRegion string) error {
 	exists, err := sqlInstanceExistsInGCP(ctx, dbInstance, gcpProject)
 	if err != nil {
 		return err
 	}
 
+	var cmd *exec.Cmd
 	if exists {
-		return nil
+		cmd = exec.CommandContext(ctx,
+			"gcloud",
+			"beta",
+			"--quiet",
+			"sql",
+			"instances",
+			"patch",
+			dbInstance,
+			"--project", gcpProject,
+			"--update-labels", fmt.Sprintf("created-by=knorten,team=%v,additional-label=value", teamSlug),
```

Comment:
I'm guessing `additional-label` is just something you used for testing?

Reply:
Yes!